src/angular/directive.js

```diff
@@ -36,6 +36,7 @@ angular.module('algolia.autocomplete', [])
         datasets: '&aaDatasets'
       },
       link: function(scope, element, attrs) {
+        if (!element.hasClass('autocomplete') && attrs.autocomplete !== '') return;
```

Comment:
Because of the `restrict` option used here: https://github.com/algolia/autocomplete.js/blob/f96a1bad72212066dab735de08d560cf6122f65b/src/angular/directive.js#L33

```js
 restrict: 'AC', // Only apply on an attribute or class
```